src/tests/test_magic_cmd.py

```diff
@@ -8,11 +8,17 @@
 @pytest.mark.parametrize(
     "cell, error_type, error_message",
     [
+        [
+            "%sqlcmd",
```

Comment:
let's test also `%sqlcmd ""`, `%sqlcmd " "`, `%sqlcmd "  "`

Reply:
Have committed the requested changes.

Reply:
Please verify.